packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -97,7 +97,7 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 		id: string,
 		runtime: IFluidDataStoreRuntime,
 		attributes: IChannelAttributes,
-		telemetryContextPrefix: string,
+		telemetryContextPrefix: string = "fluid_sharedTreeCore_",
```

Comment:
seems not meaningful to give this a different prefix than the class with policy enabled